.circleci/images/build_cci_image.sh

```diff
@@ -9,7 +9,7 @@ do
         -h|--help)
           echo "options:"
           echo "-h, --help  show brief help"
-          echo "-e, --env_type specify env_type to use from { pythn36, conda38, pyenv37, venv36 }"
+          echo "-e, --env_type specify env_type to use from { pythn36, conda38, pyenv37, venv36, conda39 }"
```

Comment:
Question : Do we still use the files in .circleci folder ?

Reply:
i didn’t find where circle ci files are used either, but i still updated and uploaded in ecr just in case.